hoa2dot/core.py

```diff
@@ -55,6 +64,10 @@ def accepting_sets(self) -> Set[int]:
     def __str__(self):
         return " ".join(map(str, self.conditions))
 
+    def __eq__(self, other):
+        """Check equality between two AtomTypes."""
```

Comment:
please fix docstring.

Reply:
isn't it ok?

Reply:
It is a method of `class And`, not `class AtomTypes`

Reply:
right!

Reply:
right